src/main/java/com/conleos/views/admin/DeleteAccountDialog.java

```diff
@@ -19,25 +20,25 @@ public class DeleteAccountDialog extends Dialog {
     public DeleteAccountDialog(Long userID) {
         User account = UserService.getInstance().getUserByID(userID);
 
-        H2 changeHeader = new H2("Delete Account forever.");
-        Text text = new Text(String.format("Are you sure you want to delete '%s' ?", account.getUsername()));
-        TextField validateField = new TextField(String.format("Enter '%s' to confirm", account.getUsername()));
+        H2 changeHeader = new H2(Lang.translate("view.deleteAccount.changeHeader"));
+        Text text = new Text(String.format(Lang.translate("view.deleteAccount.text") + " '%s' ?", account.getUsername()));
+        TextField validateField = new TextField(String.format(Lang.translate("view.deleteAccount.validateField1") + " '%s' " + Lang.translate("view.deleteAccount.validateField2"), account.getUsername()));
```

Comment:
view.deleteAccount.* should not be used.
Maybe something like 
- deleteAccountDialog.*

Reply:
All deleteAccounts are replaced with deleteAccountDialog